h2o-genmodel/src/main/java/hex/genmodel/ModelMojoReader.java

```diff
@@ -223,6 +223,23 @@ protected ModelAttributes readModelSpecificAttributes() {
     }
   }
 
+  protected void readModelKV() throws IOException {
+    readCommonModelData();
+    readModelData();
+  }
+
+  protected void readCommonModelData() throws IOException {
```

Comment:
do we need to override this somewhere? should the call just be moved to readModelKV?